Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,105 +2,57 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
-  cursor: pointer;
-  text-decoration: none !important;
-  border: none;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  padding: 0px 2px;
-  -moz-padding-start: 0;
-  -moz-padding-end: 0;
-  outline: none;
-}
-
-button.webButton::-moz-focus-inner
-{ 
-  border: 0;
+  background-color: transparent;
+  border: 1px solid var(--color-primary);
+  border-radius: 2px;
+  padding: 0.6em;
+  box-sizing: border-box;
 }
 
-button.webButton span.buttonBody
+button.webButton:hover,
+button.webButton:focus
 {
-  border: solid 1px #AAAAAA;
-  font-weight: bold;
-  background-image: url(../Image/ButtonBackground.gif);
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
-  
-  display: block;
-  position: relative;
-  top: 0px;
-  left: -2px;
-  
-  width: 100%;
-  white-space: nowrap;
-
-  padding-top: 2px;
-  padding-bottom: 2px;
-  padding-left: 1px;
-  padding-right: 1px;
+  background-color: var(--color-highlight);
 }
 
-button.webButton span.buttonBody:first-child::before
+button.webButton:focus
 {
-  content: "";
-  height: 1.25em;
-  width: 0;
-  display: inline-block;
-  vertical-align: text-bottom;
+  text-decoration: underline;
```

Comment:
Yes, you need to be able to identify the focus without relying on contrast. If you have a siginifact change in the coloring, e.g. something akin to a light/dark inversion, that works. Just using a slighlty brighter or dark color, not so much. It may be, that in the end we'll do a shadow-outline when we're done, instead of the underline, but for now, the underline is the cheapest and least controversial update to the styleguide to get the desired effect.

Reply:
The browser does a shadow outline for a button element. Since we restyle a button element we get that effect if you highlight it with tab. Then it will get a dark shadow and an underline.

One highlight should be enough right? So I would remove the underline

Reply:
Ah, that's new. IE didn't do it like that. Okay then, no underline.